packages/react-core/src/operations/aggregation.js

```diff
@@ -0,0 +1,71 @@
+import { AggregationTypes } from './constants/AggregationTypes';
+
+const dummyFn = ([value]) => value;
+
+export const aggregationFunctions = {
+  [AggregationTypes.COUNT]: (values) => values.length,
+  [AggregationTypes.MIN]: min,
+  [AggregationTypes.MAX]: max,
+  [AggregationTypes.SUM]: sum,
+  [AggregationTypes.AVG]: avg
+};
+
+export function aggregate(feature, keys, operation) {
```

Comment:
3 args already here for aggregate, have you considered using a single object param here, for consistency?